Language/Functions/USB/Keyboard/keyboardModifiers.adoc

```diff
@@ -116,5 +142,37 @@ The library can simulate function keys up to F24.
 |KEY_F24        |0xFB |251
 |===
 
+[float]
+==== Function control keys
+These are three rarely used keys that sit above the navigation cluster.
```

Comment:
```suggestion
These are three keys that sit above the navigation cluster.
```
Reduce unnecessary verbosity.

Reply:
Good point. Thanks @per1234 for your review!